plugins/producer/main.go

```diff
@@ -463,6 +456,13 @@ func getUpdates(block *gtypes.Block, td *big.Int, receipts gtypes.Receipts, dest
 }
 
 func (*cardinalProducerModule) BlockUpdates(block *gtypes.Block, td *big.Int, receipts gtypes.Receipts, destructs map[common.Hash]struct{}, accounts map[common.Hash][]byte, storage map[common.Hash]map[common.Hash][]byte, code map[common.Hash][]byte) {
+	if cfg.Cloudwatchns != "" {
```

Comment:
We only want to do this once. I'd probably wrap this with a [sync.Once](https://pkg.go.dev/sync#Once)

Reply:
On it.